src/specBuilder/scale/scaleSpecBuilder.ts

```diff
@@ -200,3 +200,36 @@ const isScaleMultiFieldsRef = (
 ): domain is ScaleMultiFieldsRef => {
 	return Boolean(domain && !Array.isArray(domain) && 'data' in domain && 'fields' in domain);
 };
+
+export const getRSCAnimationScales = (scales: Scale[]) => {
```

Comment:
I think `getRSCAnimationScales` can mislead a reader, instead can we try `addRSCAnimationScales`